src/main/target/CC3D/target.h

```diff
@@ -113,6 +113,7 @@
 //#undef USE_SERIALRX_SBUS       // Frsky and Futaba receivers
 //#undef USE_SERIALRX_IBUS       // FlySky and Turnigy receivers
 #undef USE_SERIALRX_CRSF       // Team Black Sheep Crossfire protocol
+#undef USE_SERIALRX_GHST       // ImmersionRC Ghost Protocol
```

Comment:
There is no need to modify these legacy targets - they are no longer supported.

Reply:
Mike, is there a list of obsolete legacy targets anywhere? I added the #undef to ensure that I didn't break any of them due to code size issues.

Reply:
@tonycake: `make targets` will spit out the list, and the actually supported / released targets are shown as 'Built targets':

```
Built targets:       ALIENWHOOPF7 COLIBRI_OPBL CRAZYFLIE2 CRAZYFLIE2BQ IFLIGHT_H7_TWING KISSFCV2F7 NUCLEOF446RE NUCLEOF7 NUCLEOH743 NUCLEOH743_RAMBASED REVO_OPBL SITL SPEKTRUMF400 SPRACINGH7EXTREME SPRACINGH7NANO SPRACINGH7ZERO STM32F405 STM32F411 STM32F4DISCOVERY STM32F4DISCOVERY_DEBUG STM32F745 STM32F7X2
```

Reply:
Makes sense... just one question... not removing GHST from these old legacy targets can potentially cause them to be broken. Is this acceptable? Are legacy targets still assumed to be buildable? Should they be removed from the BF sources and parked in an archive?
I am happy to remove the #undef USE_SERIALRX_GHST code that I added, but I don't understand how that can improve things.